install-from-binstall-release.sh

```diff
@@ -2,6 +2,37 @@
 
 set -euxo pipefail
 
+if [ "$OS" = "Windows_NT" ]; then
```

Comment:
This will fix the unbound variable error in CI:

```suggestion
if [ "${OS-}" = "Windows_NT" ]; then
```